Libraries/Components/TextInput/TextInput.windows.js

```diff
@@ -649,6 +649,7 @@ var TextInput = React.createClass({
           selectTextOnFocus={this.props.selectTextOnFocus}
```

Comment:
Adding support for the placeholderTextColor to the text input control.... 

This file has references to Android and iOS controls event though the file is *.windows.js   Perhaps this is left over from the split of the platforms into separate files.... Additionally, I noticed that the Windows section is split into two branches;  one for if the this.props.secureTextEntry is true and one for when it is false... When I added the placeholderTextColor property to the "true" case it did not get picked up.   This is likely because there is a separate JS for the PasswordBoxWindows.js that seems to be the controlling code for this control.    This led to some confusion...